src/SprykerEco/Zed/UnzerGui/Communication/Form/Constraint/UnzerCredentialsConstraintValidator.php

```diff
@@ -32,17 +32,17 @@ public function validate($value, Constraint $constraint): void
             ));
         }
 
-        $unzerValidationResponseTransfer = $constraint->getUnzerFacade()->validateUnzerCredentials($value);
+        $unzerCredentialsResponseTransfer = $constraint->getUnzerFacade()->validateUnzerCredentials($value);
```

Comment:
`@param \Generated\Shared\Transfer\UnzerCredentialsTransfer|string|null $value`

if it's true then add corresponding check to avoid type error on facade call here
otherwise please correct annotation